### DISCUSSION TITLE
No responses to mySQL db

### DISCUSSION THREAD

**Discussion by abhishekgoyal0306** [GrapesJS/grapesjs#4269] [General]
I cannot get a response from GrapeJS to store in MySQL DB. 
The setup is complete and I am able to use the editor on my system. Setup is on windows using GrapeJS documentation and videos.

We are implementing GrapeJS with Laravel as a page builder.
I have written the code in main.js for initializing the editor. 
I am really stuck and cannot get past this. I would be very grateful for the help. 
Here is a snippet. Any suggestions?.

const editor = grapesjs.init({
    container: "#editor",
    fromElement: true,
    width: "auto",
    storageManager: {
        autosave: true,
    },
    plugins:["gjs-preset-webpage"],
    pluginsOpts: {

    },
    StorageManager:{
        type: 'remote',
        stepsBeforeSave:3,
        autosave:true,
        autoload:true,
        contentTypeJson: true,
        storeComponents: true,
        storeStyles: true,
        storeHtml: true,
        storeCss: true,
        id: "my-",
        urlStore: "/admin/pages/${location.pathname.split('/')[2]}",
        urlLoad: "/admin/pages/${location.pathname.split('/')[2]}/edit",
        headers: {
            'Content-Type': 'application/json',
        },
    }
});

**Comment by Bishoy-Saber** [comment]
I faced the same problem while integrating with ASP.NET project and MS SQL Server DB.
If the is a solution to this problem I will be graceful if you share the code.
